Shapes.java

```diff
@@ -1,25 +1,32 @@
 import java.util.*;
+import java.lang.Math;
 
-public class Shapes{
-  public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
-
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+public class Main
```

Comment:
I think in order for this to run properly, the name of the class has to be the same as the file, so it would be `Shapes`.